packages/browser/package.json

```diff
@@ -16,7 +21,7 @@
   "scripts": {
     "dev:tsc": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
     "precommit": "lint-staged",
-    "build": "rm -rf lib/ && tsc -p tsconfig.build.json",
+    "build": "rm -rf lib/ && parcel build",
```

Comment:
nit: Is `tsconfig.build.json` still needed? Seems not

Reply:
we can use it for type check

Reply:
removed